encapsia_cli/plugins.py

```
@@ -410,6 +412,21 @@ def install(obj, versions, show_logs, latest_existing, all_available, plugins):
         lib.log("Nothing to do!")
 
 
+def variant_is_installed(api, plugin_spec):
+    response = api.run_view("pluginsmanager", "plugins", view_options={"tags": True})
+    matching_plugins = [item for item in response if item["name"] == plugin_spec.name]
+    if len(matching_plugins) == 1:
```

So you are silently ignoring the erroneous case where the result could contain *more* than one result -- granted, this should be impossible. Perhaps we should abort if it occurs, just in case.

---

it's not ignored, it will fail if there are more than one or no plugins with that name (user will get a message - agreed that it will be confusing a bit - since there are plugins with that name - but more than allowed)

---

it will display an error message but not abort the operation, if the uninstall operation fails because of it, then let the support team handle the case